.github/workflows/linux_build_test.yml

```diff
@@ -83,7 +83,7 @@ jobs:
                 matrix.geant4_version
               }}-double_down_${{
                 matrix.double_down_version
-              }}/moab:latest
+              }}:latest
```

Comment:
I'm a little concerned about this, because it means building DAGMC in a container that already has DAGMC installed....  The `moab` descriptor should be from a version of the build that stopped at MOAB